pkg/sloop/server/internal/config/config.go

```diff
@@ -172,16 +173,31 @@ func (c *SloopConfig) Validate() error {
 }
 
 func loadFromFile(filename string) *SloopConfig {
-	yamlFile, err := ioutil.ReadFile(filename)
-	if err != nil {
-		panic(fmt.Sprintf("failed to read %v. %v", filename, err))
-	}
-	var config SloopConfig
-	err = yaml.Unmarshal(yamlFile, &config)
-	if err != nil {
-		panic(fmt.Sprintf("failed to unmarshal %v. %v", filename, err))
+	if strings.Contains(filename,".yaml") {
+		yamlFile, err := ioutil.ReadFile(filename)
+		if err != nil {
+			panic(fmt.Sprintf("failed to read %v. %v", filename, err))
+		}
+		var config SloopConfig
+		err = yaml.Unmarshal(yamlFile, &config)
```

Comment:
it looks like this is the only difference between json and yaml (line 182 vs line 193),  you could put only Unmarshal inside if statement, you don't need to duplicate error handling then

Reply:
made the changes